CHANGELOG.adoc

```diff
@@ -33,7 +33,8 @@ JsonResource resource = jsonResourceFactory.createResource(uri, customOptions);
 ----
 +
 See https://github.com/eclipse-sirius/sirius-emf-json/blob/master/CHANGELOG.adoc#v250[the Sirius EMF JSON changelog] for details.
-
+- https://github.com/eclipse-sirius/sirius-web/issues/4931[#4931] [sirius-web] Add a new `updateTargetObjectId` method to `IRepresentationMetadataUpdateService`.
+Every implementation of `IRepresentationMetadataUpdateService` should add its own implementation of `updateTargetObjectId`.
```

Comment:
This does not impact anyone, the fact that a part of this code resides in a package with `api` in its path is not a good indicator of the impact of the change.